test/unit/lib/midnightSpanCorrector.js

```diff
@@ -99,6 +99,7 @@ describe('Midnight Span Corrector', () => {
       const status = openingTimes.getStatus(moment, { next: true });
       const newStatus = midnightSpanCorrector(openingTimes, status);
       momentsShouldBeSame(newStatus.nextClosed, getMoment('tuesday', 20, 0, timeZone));
+      expect(newStatus.nextOpen).to.not.be.a('string');
```

Comment:
Rather than testing for what `nextOpen` shouldn't be could this be changed to test for it being an `instanceOf(moment)`? http://chaijs.com/api/bdd/#method_instanceof